process.md

```diff
@@ -27,34 +20,47 @@ We want to avoid deincentivising authors who opt into the reproducibility challe
+
+##  Evaluation process
+
+Reviewers will asses the artefacts against a checklists provided for each ["badge" level to be awarded](https://github.com/alan-turing-institute/climate-informatics-2024-ae/blob/main/badges.md).
 
-# Submitter guidelines
+For each point, reviewers will be required to briefly note the evidence for this point, anything they have done to validate that point (e.g., what did you need to reproduce a claim), as well as the outcome (negative, neutral, or positive) and a brief reason for their judgment.
+
+## Feedback to the authors
+
+This process has been designed to be collaborative and iterative. During the first 3 weeks, reviewers will be able to feed-back to authors.
+Authors will be encouraged to push improvements to the artefact whenever possible. For example, if essential documentation is missing, then rather than submitting a negative evaluation or providing only the reviewers with the required information, the authors should provide that information via a versioned updated to the artefact.
+
+# Author submission guidelines
 
 ## What authors should provide in their artefact submission
 
-Artifact submissions require three parts:
+Artifact submissions require three parts, as described below. ==Authors will be invited to attend reviewer training, to receive full guidance on the expectations and processes for submission described below.==
 
 1. An overview of the artifact.
-2. A non-institutional URL pointing to either: a single file containing the artifact (recommended), or
-the address of a public source control repository.
-3. A hash certifying the version of the artifact at submission time: either an md5 hash of the single file (use the md5 or md5sum command-line tool to generate the hash), or the full commit hash for the repository (e.g., from git reflog --no-abbrev).
 
-The URL must be non-institutional to protect the anonymity of reviewers (in the case of a _closed_ proccess). Acceptable URLs can be obtained from Google Drive, Dropbox, Gitlab, Zenodo, and many other providers.
+2. A non-institutional URL pointing to either: a single file containing the artifact (recommended), or the address of a public source control repository. The URL must be non-institutional to protect the anonymity of reviewers. Acceptable URLs can be obtained from Google Drive, Dropbox, Gitlab, Zenodo, and other providers. URLs may be generated using reviewer-only links. ==Do we prefer a versioned URL (e.g. DOI) here, or is that only expected if "accessible"==
```

Comment:
```suggestion
2. A non-institutional URL pointing to either: a single file containing the artefact (recommended), or the address of a public source control repository. The URL must be non-institutional to protect the anonymity of reviewers. Acceptable URLs can be obtained from Google Drive, Dropbox, Gitlab, Zenodo, and other providers. URLs may be generated using reviewer-only links. ==Do we prefer a versioned URL (e.g. DOI) here, or is that only expected if "accessible"==
```

Reply:
I suggest we avoid recommending Google Drive and/or Dropbox. Instead, we should promote accepted version control platforms for software and data.